frontend-service/src/test/java/com/amazonaws/blox/frontend/operations/CreateEnvironmentTest.java

```diff
@@ -51,26 +48,8 @@ public void mapsInputsAndOutputsCorrectly() throws Exception {
         .thenReturn(
             com.amazonaws.blox.dataservicemodel.v1.model.wrappers.CreateEnvironmentResponse
                 .builder()
-                .environment(
-                    Environment.builder()
-                        .environmentId(id)
-                        .role(ROLE)
-                        .environmentType(ENVIRONMENT_TYPE)
-                        .environmentHealth(HEALTHY)
-                        .environmentStatus(STATUS)
-                        .deploymentMethod(DEPLOYMENT_METHOD)
-                        .deploymentConfiguration(deploymentConfiguration)
-                        .createdTime(Instant.now())
-                        .lastUpdatedTime(Instant.now())
-                        .build())
-                .environmentRevision(
-                    EnvironmentRevision.builder()
-                        .environmentId(id)
-                        .environmentRevisionId(ENVIRONMENT_REVISION_ID)
-                        .instanceGroup(instanceGroup)
-                        .taskDefinition(TASK_DEFINITION)
-                        .createdTime(Instant.now())
-                        .build())
+                .environment(environmentDS(id, deploymentConfiguration))
```

Comment:
Not in the scope of this PR, but I think we can replace the construction of Environment/EnvironmentRevision with the ones from EnvironmentControllerTestCase in other FE operation tests as well.

Reply:
Agreed. I started down that route, and figured we'd want to use the ModelBuilders from the DS integ tests instead.